at_client/src/main/java/org/atsign/client/util/RegisterUtil.java

```diff
@@ -108,34 +109,53 @@ public Boolean registerAtsign(String email, AtSign atsign, String registrarUrl,
      * @throws IOException
```

Comment:
Please add documentation explaining use of the new 'confirmation' parameter (which should be optional as mentioned in next comment)